### PR TITLE
fix(demo): suggest link

### PR DESCRIPTION
**Pull request by BrocksiNet**
### Description

Closes https://github.com/shopware/frontends/issues/257

### Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

### Additional context

I removed the localPath from search route. I guess this will not be translated.
FYI @mdanilowicz.

### CODE REVIEW

**Review by mdanilowicz — APPROVED** [review]

**Review by mdanilowicz — CHANGES_REQUESTED** [review]

File: templates/vue-demo-store/components/layout/LayoutStoreSearch.vue [hunk 1]
```diff
@@ -119,7 +119,7 @@ watch(enter, (value) => {
         <div v-else>
           <NuxtLink
             v-if="getTotal > 0"
-            :to="localePath(`/search?query=${typingQuery}`)"
+            :to="`/search?query=${typingQuery}`"
```
[the remark this answers is not in the document]
Comment (BrocksiNet) [reply templates/vue-demo-store/components/layout/LayoutStoreSearch.vue]: Correct. Updated. Good point. Thx.

File: templates/vue-demo-store/components/layout/LayoutStoreSearch.vue [hunk 2]
```diff
@@ -119,7 +119,7 @@ watch(enter, (value) => {
         <div v-else>
           <NuxtLink
             v-if="getTotal > 0"
-            :to="localePath(`/search?query=${typingQuery}`)"
+            :to="localePath(`/search`) + `?query=${typingQuery}`"
```
Comment (patzick) [inline templates/vue-demo-store/components/layout/LayoutStoreSearch.vue:122]: what about something like this?
```
:to="localePath({path: `/search`, query: {query: typingQuery})"
```

Comment (BrocksiNet) [reply templates/vue-demo-store/components/layout/LayoutStoreSearch.vue]: Works and looks nicer. :green_apple:

**Review by patzick — APPROVED** [review]
👌 💯

**Pull request merged**